test/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAnAssembly.cs

```diff
@@ -26,6 +26,13 @@ public class GivenThatWeWantToReferenceAnAssembly : SdkTest
             string referencerTarget,
             string dependencyTarget)
         {
+            if (UsingFullFrameworkMSBuild)
```

Comment:
So then why not just leave these tests alone?  Again, changing these tests seems irrelevant to your current work.